src/index.js

```diff
@@ -10,7 +10,7 @@ const Channel = require('chnl');
 const PromiseController = require('promise-controller');
 const Requests = require('./requests');
 const defaultOptions = require('./options');
-const {throwIf} = require('./utils');
+const { throwIf } = require('./utils');
```

Comment:
Could you please keep codestyle.

Reply:
sorry, didn't notice this one - it is auto from the rules we apply internally :)